CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner might be
+# required for merging a PR (depends on repository settings).
+#
+# For details about syntax, see:
+# https://help.github.com/en/articles/about-code-owners
+# But here are some important notes:
+#
+# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
+#   which can be everywhere.
+# - Multiple owners are supported.
+# - Either handle (e.g, @github_user or @github_org/team) or email can be used. Keep in mind,
+#   that handles might work better because they are more recognizable on GitHub,
+#   you can use them for mentioning unlike an email.
+# - The latest matching rule, if multiple, takes precedence.
+
+# CI
+/.github/ @paritytech/ci
```

Comment:
This folder does not exist yet. How about creating it and moving this file there? Github supports the `.github/CODEOWNERS` location according to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location.

Reply:
Initial idea of setting up CODEOWNERS was connected with [#187](https://github.com/paritytech/ci_cd/issues/187) so we would like to have them in the tool (GitHub, GitLab) independent place.
Also AFAIK there is no problem to have non-existent paths in CODEOWNERS file. But the preference is if/when some GitHub Action's stuff will be added - CI team will be informed.

Reply:
Moving this file to `/.github` means the CI team would manage it according to this rule, so it seems like a better idea.

Reply:
When we discussed where to place CODEOWNERS file, we decided that most suitable place is the root of repositories. So it can be equally processed by GitHub and GitLab.
And it is already done that way for around 30 repos. So for uniformity reason it is better keep it where it is.